static/scenes/railroad/js/present-system.js

```diff
@@ -19,8 +19,11 @@ class PresentSystem {
     constructor(placeholderScene) {
         this.loader = new THREE.OBJLoader();
         this.placeholderScene = placeholderScene;
-        const material0 = new THREE.MeshToonMaterial( {color: 0xF9D231} ); 
-        const material1 = new THREE.MeshToonMaterial( {color: 0x009BFF});
+        const material0 = new THREE.MeshToonMaterial( {color: 0xF9D231}); 
```

Comment:
Consider also using the colors from our style guide: https://docs.google.com/presentation/d/1-Uz6e5t-Gvs8H2MLX1u4bG5KZqBGUB9KewtbmoteG10/view#slide=id.g628d25aa90_0_311

(I see the original present didn't do that)